pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/AvgAggregationFunction.java

```diff
@@ -157,6 +239,9 @@ public AvgPair extractAggregationResult(AggregationResultHolder aggregationResul
 
   @Override
   public AvgPair extractGroupByResult(GroupByResultHolder groupByResultHolder, int groupKey) {
```

Comment:
`extractAggregationResult()` also needs to be modified to return `null` when null handling is enabled